src/main/java/salesforce/utils/StringDateConverter.java

```diff
@@ -0,0 +1,116 @@
+package salesforce.utils;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.Locale;
+
+public class StringDateConverter {
+
+    private static LocalDateTime date;
+    private static final String SIMPLE_DATE_FORMAT = "\\d{4}(-|/)(0[1-9]|1[012])(-|/)(0[1-9]|[12][0-9]|[3][01])";
+    private static final String SIMPLE_TIME_FORMAT = "T\\d{2}:\\d{2}:\\d{2}";
+    private static final String SINGLE_WORD_FORMAT = "[ADEORSMWYT]+";
+    private static final String ENUM_FORMAT = "[\\d]+\\s((day|days)|(month|months)|(year|years)|(seconds|second)"
+            + "|(minutes|minute)|(hour|hours))\\s(from now|ago)";
+
+    /**
+     * Sends the String to the method to convert.
+     * @param dateString the String date to convert.
```

Comment:
You should add a newLine to match the checkstyle convention.

Reply:
done